modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartDioNextClientCodegen.java

```diff
@@ -278,78 +279,224 @@ public void postProcessModelProperty(CodegenModel model, CodegenProperty propert
+        }
+        return typeData;
+    }
+
+    //Extracts inline enums to put them later in an "inlineEnums" property for each api
+    private void extractInlineEnums(IJsonSchemaValidationProperties property, Map<String, Object> inlineEnums) {
+        if (property instanceof CodegenProperty) {
+            final CodegenProperty casted = (CodegenProperty) property;
+            if (casted.isEnum) {
+                if (!inlineEnums.containsKey(casted.enumName)) {
+                    inlineEnums.put(casted.enumName, casted);                 
+                }
+            }
+        } else if (property instanceof CodegenParameter) {
+            final CodegenParameter casted = (CodegenParameter) property;
+            if (casted.isEnum) {
+                if (!inlineEnums.containsKey(casted.enumName)) {
+                    inlineEnums.put(casted.enumName, casted);                 
+                }
+            }
+        }
+        final CodegenProperty items = property.getItems();
+        if (items != null) {
+            if (items.isContainer) {
+                extractInlineEnums(items.items, inlineEnums);
+            } else if (items.isEnum) {
+                extractInlineEnums(items, inlineEnums);
+            }
+        }
+    }
     @Override
     public Map<String, Object> postProcessOperationsWithModels(Map<String, Object> objs, List<Object> allModels) {
         super.postProcessOperationsWithModels(objs, allModels);
         Map<String, Object> operations = (Map<String, Object>) objs.get("operations");
         List<CodegenOperation> operationList = (List<CodegenOperation>) operations.get("operation");
 
         Set<Map<String, Object>> serializers = new HashSet<>();
+        Map<String, Object> inlineEnums = new HashMap<>();
         Set<String> resultImports = new HashSet<>();
-
-        for (CodegenOperation op : operationList) {
-            for (CodegenParameter param : op.allParams) {
-                if (((op.isMultipart && param.isFormParam) || param.isBodyParam) && (param.isBinary || param.isFile)) {
-                    param.dataType = param.dataType.replace("Uint8List", "MultipartFile");
-                    param.baseType = param.baseType.replace("Uint8List", "MultipartFile");
-                    op.imports.add("MultipartFile");
-
-                    if (SERIALIZATION_LIBRARY_BUILT_VALUE.equals(library)) {
-                        boolean skipFormModel = Boolean.parseBoolean(GlobalSettings.getProperty(CodegenConstants.SKIP_FORM_MODEL, "true"));
-                        if (param.isFormParam && param.isContainer && !skipFormModel) {
-                            // Because of skipFormModel=false, there is a model class generated which has
-                            // "BuiltList<Uint8List>" as property and it requires the correct
-                            // serializer imports to be added in order to compile.
-                            addBuiltValueSerializerImport("Uint8List");
```

Comment:
This whole block is now missing, we don't have a sample for `skipFormModel` but I don't think it will compile atm.